docs/guidance/DataManagementAnalytics-LimitingCrossTenantPrivateEndpointProvisioning.md

```diff
@@ -234,7 +234,7 @@ Azure Synapse also uses managed virtual networks and therefore a similar policy
 }
 ```
 
-The policy above enforces the use of the data exfiltration feature of Synapse. Synapse also allows to deny any private endpoint that is coming from a service that is hosted outside of the customer tenant or a specified set of tenant ids. The policy above enforces exactly that and only allows the creation of managed private endpoints that are linked to services that are hosted in the customer tenant.
+The policy above enforces the use of the data exfiltration feature of Synapse. Synapse also allows to deny any private endpoint that is coming from a service that is hosted outside of the customer tenant or a specified set of tenant IDs. The policy above enforces exactly that and only allows the creation of managed private endpoints that are linked to services that are hosted in the customer tenant.
```

Comment:
Clarification - "Synapse also allows to deny any private endpoint" (?)
Did you mean "Synapse denies any private endpoint coming from a service hosted outside of customer's tenant ... " ?

Reply:
Not, if you don't set it up correctly. This is correctly described above.